en/05_Contributing/04_Release_Process.md

```diff
@@ -179,6 +179,12 @@ When deprecating config:
 
 * Add the following docblock `@deprecated 1.2.3 Use different_config instead`
 
+When deprecating some behaviour or combination of configuration values:
+
+* Add `Deprecation::notice('1.2.3', 'Using x with y is deprecated. Do [other thing] instead', SCOPE_GLOBAL);`
```

Comment:
Oops this should be `Deprecation::SCOPE_GLOBAL` and the above should be `Deprecation::SCOPE_CLASS` - I'll fix that.